app/views/layouts/application.html.haml

```diff
@@ -7,6 +7,8 @@
     = csrf_meta_tags
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %meta{:content => "IE=edge,chrome=1", "http-equiv" => "X-UA-Compatible"}/
```

Comment:
What a mess! Feature doesn't belong to this PR and additionally this commit contains another files which are not belong to the commit too. Please don't do this again. It's horrible :)

Reply:
Ok sorry I should do it separate :(

Reply:
:) Merge this PR.

Reply:
@nikos83 I wrote how manage conflicts in the first doc.